taskcluster/ci/build/kind.yml

```diff
@@ -165,7 +165,7 @@ jobs:
 
     beta:
         attributes:
-            release-type: beta
+            release_type: beta
```

Comment:
Hmm so I don't think this actually matches gecko?  In gecko `release_type` is a decision task parameter, as far as I can tell, not a task attribute.  So I'm not sure it's worth doing this renaming here for that reason.

Reply:
You're right. That was some bad wording. What I meant was: in gecko, attributes use underscores instead of dashes. I'll change the commit message to make it less ambiguous.